app.js

```diff
@@ -6,20 +6,30 @@
 // a number of minutes as an argument and returns
 // the number of seconds
 
-function convertToSeconds() {}
-
+function convertToSeconds(minutes){
+    return minutes * 60;
+}
 console.log(`convertToSeconds(10) === 600 : ${convertToSeconds(10) === 600}`);
 
 // 2. areaOfTriangle
 // Write a function called areaOfTriangle that takes in the base and height
 // of a triangle and returns its area
-function areaOfTriangle() {}
+function areaOfTriangle(base, height) {
+    return base + height + height; 
```

Comment:
while this technically works for the given console.log, the function that will generalize to get the area of all triangles returns (base * height) / 2